vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt

```diff
@@ -27,11 +27,7 @@ internal object JsonCredentialSerializer {
 }
 
 val jsonSerializer by lazy {
-    Json {
-        prettyPrint = false
-        encodeDefaults = false
-        classDiscriminator = "type"
-        ignoreUnknownKeys = true
+    Json(from = at.asitplus.crypto.datatypes.jws.io.jsonSerializer) {
```

Comment:
I would not remove the explicit configuration of the serializer in this library

Reply:
Makes sense fro JSON